libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -389,7 +389,7 @@ void RangeFinder::detect_instance(uint8_t instance, uint8_t& serial_instance)
     case Type::BenewakeTFminiPlus:
         FOREACH_I2C(i) {
             if (_add_backend(AP_RangeFinder_Benewake_TFMiniPlus::detect(state[instance], params[instance],
-                                                                        hal.i2c_mgr->get_device(i, params[instance].address)))) {
+                                                                        hal.i2c_mgr->get_device(i, params[instance].address==-1?0x10:params[instance].address)))) {
```

Comment:
RNGFND1_ADDR's range is 0-127 because it's AP_Int8.
https://ardupilot.org/copter/docs/parameters.html#rngfnd1-addr-bus-address-of-sensor

Reply:
@tatsuy san. Comments thanks.
I have an I2C address defined in the I2C specification.
ArduPilot overlooks this definition.
I think that a value that cannot be used as an address, -1, is better to make the address of 0 valid.
The range finder uses the value of ADDRESS when the value of TYPE is other than 0.

ex: page 17.
https://www.nxp.com/docs/en/user-guide/UM10204.pdf